.hlint.yaml

```diff
@@ -27,3 +27,4 @@
 - ignore: {name: Use first, within: [UntypedPlutusCore.Evaluation.Machine.Cek]}
 - ignore: {name: Redundant if, within: [PlutusLedgerApi.V1.Value, PlutusLedgerApi.V1.Data.Value]}
 - ignore: {name: Replace case with maybe, within: [PlutusLedgerApi.V1.Value, PlutusLedgerApi.V1.Data.Value]}
+- ignore: {name: Use bimap, within: [PlutusTx.Builtins.HasOpaque]}
```

Comment:
Oh no no no no no, not this fuckery again. If we make it mandatory or it is already mandatory, I will not comply.

Reply:
I don't like ignoring HLint warnings in a place that's far away from the actual warning, but this is better for me than not doing anything, as HLint warnings interfere with the legit warnings and break my workflow.
What's your take on HLint warnings?

Reply:
Can you instruct HLint to use your own config file instead of the one in the repo?

Reply:
> What's your take on HLint warnings?

I hate HLint and don't use it. I do not mind you using it of course.

> Can you instruct HLint to use your own config file instead of the one in the repo?

Apparently there are two in the repo. One for CI and one for I don't really know what.